pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -1,10 +1,24 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
+, substituteAll
 , cmake
+, openjpeg
 , libyamlcpp
-, git
+, batchVersion ? false
+, withJpegLs ? true
+, withOpenJpeg ? true
+, withCloudflareZlib ? true
 }:
 
+let
+  cloudflareZlib = fetchFromGitHub {
+    owner = "ningfei";
+    repo = "zlib";
+    rev = "gcc.amd64";
+    sha256 = "sha256-qySFwY0VI2BQLO2XoCZeYshXEDnHh6SmJ3MvcBUROWU=";
+  };
+in
```

Comment:
@neurolabusc, we have to fetch these sources outside of CMake because the build environment is isolated from the network. NixOS would consider a network request during the build process "impure."